components/director/internal/domain/application/repository_test.go

```diff
@@ -991,6 +991,149 @@ func TestPgRepository_GetBySystemNumber(t *testing.T) {
 	suite.Run(t)
 }
 
+func TestPgRepository_ListByLocalTenantID_NoFilter(t *testing.T) {
+	localTenantID := "local-tenant-id"
```

Comment:
I am not sure why the tests are passing, because this `localTenantID` value, which is passed to the query here - `...WHERE (local_tenant_id = $1...` is different from the application's `localTenantID` value which is returned. 
Passed `localTenantID` value - `local-tenant-id`
Applicaiton's `localTenantID` value - `local-tenant-id-123`

Reply:
I think the same is valid and for the other repo layer unit tests and probably for the service tests as well.